mne/tests/test_dipole.py

```diff
@@ -26,6 +26,7 @@
 from mne.surface import _compute_nearest
 from mne.bem import _bem_find_surface, read_bem_solution
 from mne.transforms import apply_trans, _get_trans
+from mne.beamformer import rap_music
```

Comment:
don't use rap_music in a test that is unrelated to rap_music. Just add a print statement in a test that has a DipoleFixed object.